alerta/auth/basic_ldap.py

```diff
@@ -0,0 +1,72 @@
+
```

Comment:
This file is more or less a stripped down version of alerta/auth/base.py. 
Email-verification, password-hash checking and signup have been removed.